src/components/icons/index.tsx

```diff
@@ -0,0 +1,3 @@
+import { IAIcon } from "./IAIcon";
+
+export const Icons = { IA: IAIcon };
```

Comment:
🤨 
```suggestion
export const Icons = { AI: IAIcon };
```

Reply:
There are multiple brand icons around the designs; we will need to create a dropdown selector for available icons instead of letting the user update each one.

This is just a placeholder; I will continue working on it in this ticket: https://dept-us.atlassian.net/browse/RT-100